cmd/root.go

```diff
@@ -44,6 +44,7 @@ func init() {
 		StringVarP(&envName, "env", "e", "", "Environment name, syntax(type.env.platform)")
 	rootCmd.PersistentFlags().
 		BoolVar(&isDebug, "debug", false, "debug mode")
+	rootCmd.MarkPersistentFlagRequired("env")
```

Comment:
env is now mandatory, any reason ?

Reply:
It is needed for all our command. It allows us to skip adding a `PreRunE` and check if `env` is filled :)

Reply:
if then env is not provided by default jarvis used to iterate over all envs no ?

Reply:
Yes you are right. But it is not documented and not sure if it is the good way to do it.
I don't see the use case. Do you see one? I think if we have one, we must be explicit on that case.
`--env all` ?

Reply:
I see one usecase for `ansible run`, it's handy to run a command over all envs.

I like the idea to be explicit `--env all`

Reply:
I propose to let the `MarkRequired...` for this PR and adding the `--env all` as an issue?

Reply:
k